src/poketch_data.c

```diff
@@ -236,73 +236,72 @@ void PoketchData_MapMarkerPos(const PoketchData *poketchData, int index, u8 *x,
     *y = poketchData->markMapPositions[index].y;
 }
 
-BOOL sub_02056A10 (const PoketchData *poketchData)
+BOOL PoketchData_DotArtModified(const PoketchData *poketchData)
 {
-    return poketchData->unk_00_2;
+    return poketchData->dotArtModifiedByPlayer;
 }
 
-void sub_02056A18 (const PoketchData *poketchData, u8 *param1)
+void PoketchData_DotArtData(const PoketchData *poketchData, u8 *dst)
 {
-    if (poketchData->unk_00_2) {
-        MI_CpuCopy8(poketchData->unk_2A, param1, 120);
+    if (poketchData->dotArtModifiedByPlayer) {
+        MI_CpuCopy8(poketchData->dotArtData, dst, POKETCH_DOTART_SIZE_BYTES);
     }
 }
 
-void sub_02056A2C (PoketchData *poketchData, const u8 *param1)
+void PoketchData_SetDotArtData(PoketchData *poketchData, const u8 *src)
 {
-    MI_CpuCopy8(param1, poketchData->unk_2A, 120);
-    poketchData->unk_00_2 = 1;
+    MI_CpuCopy8(src, poketchData->dotArtData, POKETCH_DOTART_SIZE_BYTES);
+    poketchData->dotArtModifiedByPlayer = TRUE;
 }
 
-void sub_02056A48 (PoketchData *poketchData, const BoxPokemon * param1)
+void PoketchData_PokemonHistoryAddEntry(PoketchData *poketchData, const BoxPokemon * boxPokemon)
 {
-    int v0;
+    int new_pokemon_index = PoketchData_PokemonHistorySize(poketchData);
```

Comment:
I think you can just name this `idx` or `next`, and the meaning is perfectly clear.